src/screens/Instructors/screens/Instructor/index.js

```diff
@@ -10,14 +11,20 @@ export default ({instructor}) => (
       items={[
         <div>
           <InstructorInfoCard instructor={instructor} />
-          <InstructorStatsCard
-            publishedLessons={instructor.published_lessons}
-            publishedCourses={instructor.published_courses}
-          />
         </div>,
         <InstructorLessonsCard instructor={instructor} />,
       ]} 
       relativeSizes={[1, 2]}
     />
+    <LayoutColumns
+      items={[
+        <InstructorStatsCard
+          publishedLessons={instructor.published_lessons}
+          publishedCourses={instructor.published_courses}
+        />,
+        <InstructorRevenueCard revenueUrl={instructor.revenue_url} />,
+      ]}
+      relativeSizes={[1, 2]}
+    />
```

Comment:
Adds revenue to `Instructor` screen if you have access to view it:
![image](https://cloud.githubusercontent.com/assets/5497885/24528297/14554374-1563-11e7-99fe-676ab3faf81f.png)